CMake/vtkMacroKitPythonWrap.cmake

```diff
@@ -308,7 +308,7 @@ macro(vtkMacroKitPythonWrap)
         )
     else (APPLE)
       install(TARGETS ${MY_KIT_NAME}Python
-        EXPORT VMTK-Targets
+	#EXPORT VMTK-Targets
```

Comment:
Why did you comment this line?

Reply:
It’s borrowed from PR #340 to comply with newest version of cmake and vtk 8.2.

Reply:
I just merged #340